packages/experimental/ExperimentalIcon/src/SvgIcon/SvgIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { View } from 'react-native';
+import { SvgUri } from 'react-native-svg';
+import { mergeProps, stagedComponent } from '@fluentui-react-native/framework';
+import { svgIconName, SvgIconProps } from './SvgIcon.types';
+import { useSvgIcon } from './useSvgIcon';
+
+export const SvgIcon = stagedComponent((props: SvgIconProps) => {
+  const svgProps = useSvgIcon(props);
+  return (final: SvgIconProps) => {
+    const { style, height, width, src, uri, viewBox, color, ...rest } = mergeProps<SvgIconProps>(svgProps, final);
+
+    return src || uri ? (
+      <View style={style} accessibilityRole="image" {...rest}>
+        {src && src({ viewBox, width, height, color })}
```

Comment:
The new object being used to create the object passed into src probably isn't rerender friendly.

Reply:
I know, but there was an error when I used it as <src {...props} />. "Property 'src' does not exist on type 'JSX.IntrinsicElements'." Though if I use `final.src` there's no error

Reply:
This is probably a place for memoization then.

Reply:
Oh nvm I see what you did. That works.